src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -1157,7 +1157,7 @@ public Invite createInvite(JSONObject object)
             if (usernameArray == null)
                 usernames = null;
             else
-                usernames = Collections.unmodifiableList(StreamSupport.stream(usernameArray.spliterator(), false).map(String::valueOf).collect(Collectors.toList()));
+                usernames = Collections.unmodifiableList(StreamSupport.stream(usernameArray.spliterator(), false).map(json -> new JSONObject(String.valueOf(json)).getString("username")).collect(Collectors.toList()));
```

Comment:
```suggestion
                usernames = map(channelObject, "recipients", (json) -> json.getString("username"));
```

Reply:
yup, and the `usernameArray` variable can be removed, together with the `if` (the `map` function already checks for existence)